rclcpp/include/rclcpp/service.hpp

```diff
@@ -137,7 +137,7 @@ class Service : public ServiceBase
     return std::shared_ptr<void>(new rmw_request_id_t);
   }
 
-  void handle_request(std::shared_ptr<void> & request_header, std::shared_ptr<void> & request)
+  void handle_request(std::shared_ptr<void> request_header, std::shared_ptr<void> request)
```

Comment:
Question unrelated to this PR: since this method is defined in the templated class why can't the signature use `shared_ptr` of the specific types (`ServiceT::Request`, `rmw_request_id_t`) instead of `void`?